src/code.rs

```diff
@@ -419,22 +420,16 @@ fn build_table_fns(
 
     // template variables
     let table_name = table.name.to_string();
-    #[cfg(feature = "async")]
-    let async_keyword = if table_options.get_async() {
-        " async"
-    } else {
-        ""
-    };
-    #[cfg(not(feature = "async"))]
-    let async_keyword = "";
-    #[cfg(feature = "async")]
-    let await_keyword = if table_options.get_async() {
-        ".await"
-    } else {
-        ""
+    #[allow(unused_labels)] // label is only used if feature is enabled
+    let (async_keyword, await_keyword) = 'async_block: {
```

Comment:
I'm not a fan of this syntax. Would it be easier to read if we extracted this to a function instead of using a label?

Reply:
we could extract it out to a function, i just wanted it to be in-line like the previous version, just with less repeats of `cfg` and rust does not allow `return`s from blocks, only breaks (and i labeled it to be explicit)

(and i didnt do a in-line function because i thought a function would be unnecessary for this case, especially because the previous one didnt need one)